src/server/components/lists/controllers/ingest/ingestPutController.ts

```diff
@@ -10,13 +10,22 @@ import { deserialise } from "server/models/listItem/listItemCreateInputFromWebho
 import { getServiceTypeName } from "server/components/lists/helpers";
 import {EVENTS} from "server/models/listItem/listItemEvent";
 
+interface FormData {
+  name: string,
+  fees: string,
+  questions: Array<Record<string, string | number | Array<Record<string, string | boolean>>>>,
+  metadata: Record<string, string | boolean>,
+}
+
 export async function ingestPutController(
   req: Request,
   res: Response
 ): Promise<void> {
   const id = req.params.id;
   const serviceType = getServiceTypeName(req.params.serviceType) as ServiceType;
-  const { value, error } = formRunnerPostRequestSchema.validate(req.body);
+  const fromAnnualReview = (req.body?.name?.includes("annual-review"));
+  const bodyData = fromAnnualReview ? addDeclarationData(req.body) : req.body;
```

Comment:
I don't think `addDeclarationData` is needed anymore, since we're keeping the current `declaration` page (rather than use the default `declaration`)

Reply:
Good point 👍🏿